docs/content/guides/dagster-pipes/dagster-pipes-details-and-customization.mdx

```diff
@@ -156,19 +165,31 @@ height={1313}
 
 ---
 
-## Pipes session lifecycle (orchestration process)
+## Pipes session
 
-A **pipes session** is the time spanning:
+A **Pipes session** is the time spanning:
 
 1. The creation of communications channels between the orchestration and external process;
 2. The launching and terminating of the external process;
 3. The processing of all messages reported by the external process and the closing of communications channels.
 
-Pipes sessions are created in the orchestration process with the <PyObject object="open_pipes_session" /> context manager. This function is exported by `dagster` and should be called inside of an asset or op compute function - somewhere where an <PyObject object="OpExecutionContext" /> or <PyObject object="AssetExecutionContext" /> is available:
+There are separate APIs for interacting with a Pipes session in the orchestration and external processes. For the orchestration process, these APIs are defined in `dagster`. For the external process, these APIs are defined by a Pipes integration library that is loaded by user code in the external process. This library knows how to interpret the bootstrap payload and spin up a context loader and message writer.
```

Comment:
I'm a bit confused by this. 

"For the external process, these APIs are defined by a Pipes integration library that is loaded by user code in the external process."

Do you mean just `dagster-pipes`?

Reply:
It was a bit confusing, since we don't have any additional libraries right now.